CourseNetworkAPI/CourseNetworkAPI.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Nice. I like aspnetcore so much more. I was thinking we should combine the existing api and this guy. Easier to deploy. Can we do that? Doesn’t have to be you and doesn’t have to be this PR. Maybe you can show Iswarya